HIP/hip-226.md

```diff
@@ -87,13 +89,16 @@ The following JSON represents a typical response result from either of these cal
     {
       "address": "0x0000000000000000000000000000000000001f42",
       "bloom": "0x8f705727c88764031b98fc32c314f8f9e463fb62",
+      "contract_id": "0.0.1003",
       "data": "0x1513001083c899b1996ec7fa33621e2c340203f0",
+      "index": 1,
       "topics": [
         "0xaf846d22986843e3d25981b94ce181adc556b334ccfdd8225762d7f709841df0",
         "0x0000000000000000000000000000000000000000000000000000000000000765"
       ]
     }
   ],
+  "result": "SUCCESS",
   "state_changes": [
     {
       "after": "0xaf846d22986843e3d25981b94ce181adc556b334ccfdd8225762d7f709841df0",
```

Comment:
Update state_changes to reflect latest design.

Reply:
Updated